shared/js/background/safari-wrapper.es6.js

```diff
@@ -1,6 +1,4 @@
-/* global safari:false tabManager:false */
-// const tabManager = require('./tab-manager.es6')
```

Comment:
Turned out this was getting messed up because of a circular dependency. I'm sure we've got these elsewhere but ah well.

Reply:
That makes sense now. Nice find.